sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Azure.Messaging.EventHubs.Processor.csproj

```diff
@@ -22,6 +22,9 @@
     <PackageReference Include="System.Threading.Tasks.Extensions" />
   </ItemGroup>
 
+  <!-- Import the Azure.Core reference -->
+  <Import Project="$(MSBuildThisFileDirectory)..\..\..\core\Azure.Core\src\Azure.Core.props" />
```

Comment:
Does this work if Azure.Messaging.EventHubs is using Experimental?

Reply:
It was passing test runs after your merge, so I believe so.   Experimental pulls Core in as a transient dependency, so it's not necessary but so long as the versions don't conflict, it won't hurt.   This particular change was just a reformat as I realized that having this separated from the other references was annoying.  😄